build.gradle

```diff
@@ -391,29 +398,27 @@ project(':core') {
   }
 
   checkstyle {
```

Comment:
No, this is "re-configuring" the configProperties to use a special import control file since core is under a separate package structure and you can't have 2 package structures in a single import control file.